lib/Cpdf.php

```diff
@@ -6249,7 +6249,7 @@ function addJpegFromFile($img, $x, $y, $w = 0, $h = 0)
             $imageHeight = $this->imagelist[$img]['h'];
             $channels = $this->imagelist[$img]['c'];
         } else {
-            $tmp = getimagesize($img);
+            $tmp = @getimagesize($img);
```

Comment:
If the image is not readable here `$tmp` is false and the following two lines will generate warnings since you can't access an offset of a boolean.

Reply:
This is a sample image that created the error
https://drive.google.com/file/d/1PSQKGyUVyj99I7zS-E9lLmdRtrrbQ1Ic/view?usp=drive_link

Reply:
You'll need to give me permission to access the file.

Reply:
Yes, I have granted access.

Reply:
Thanks, I can reproduce with that image. That image is problematic but can still be processed by GD. So, your change works for this particular case. We should still tweak the logic in case the image can't be processed at all. I think all we need to do is immediately return if the result is false:

```suggestion
            $tmp = @getimagesize($img);
            if ($tmp === false) {
                return;
            }
```